include/aws/io/tls_channel_handler.h

```diff
@@ -356,17 +468,9 @@ struct aws_tls_ctx_pkcs11_options {
      */
     struct aws_byte_cursor private_key_object_label;
 
-    /**
-     * Certificate's file path on disk (UTF-8).
-     * The certificate must be PEM formatted and UTF-8 encoded.
-     * Zero out if passing in certificate by some other means (such as file contents).
-     */
-    struct aws_byte_cursor cert_file_path;
-
     /**
```

Comment:
you can't remove this, this is something users might set directly. it would break the API